core/src/OpenMPTarget/Kokkos_OpenMPTarget_Exec.cpp

```diff
@@ -74,6 +74,8 @@ int64_t OpenMPTargetExec::m_scratch_size      = 0;
 int* OpenMPTargetExec::m_lock_array           = nullptr;
 int64_t OpenMPTargetExec::m_lock_size         = 0;
 uint32_t* OpenMPTargetExec::m_uniquetoken_ptr = nullptr;
+int OpenMPTargetExec::MAX_ACTIVE_THREADS =
+    Kokkos::Experimental::OpenMPTarget().concurrency();
```

Comment:
This is barely legal.
It violates Kokkos semantics (building an execution space before Kokkos is initialized) and just happens to work with the current OpenMPTarget implementation.

Reply:
I will move this into one of the initialize routines.